VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts

```diff
@@ -3,19 +3,24 @@ import { Subscription } from 'rxjs';
 import { VideoWebService } from 'src/app/services/api/video-web.service';
 import { EventsService } from 'src/app/services/events.service';
 import { Logger } from 'src/app/services/logging/logger-base';
+import { Hearing } from '../../shared/models/hearing';
+import { InstantMessage } from '../../services/models/instant-message';
 
 @Component({
     selector: 'app-unread-messages',
     templateUrl: './unread-messages.component.html',
     styleUrls: ['./unread-messages.component.scss']
 })
 export class UnreadMessagesComponent implements OnInit, OnDestroy {
-    @Input() conferenceId: string;
+    @Input() hearing: Hearing;
+
     messagesSubscription$: Subscription = new Subscription();
     unreadCount: number;
-    constructor(private videoWebService: VideoWebService, private eventsService: EventsService, private logger: Logger) {}
+    conferenceId: string;
```

Comment:
Do not need this property. ConferenceId should be used from the hearing object

Reply:
done